### PR TITLE
New Features

### PR DESCRIPTION
**Pull request by VMFenix8**
This is the home page that i sent in the discord

### CODE REVIEW

**Comment by esarajyan** [comment]
Remove the `dist` folder. It contains the build of the project, which can be regenerated anytime. It is usually not recommended to push to the repo.

**Comment by esarajyan** [comment]
Space needs to be added here 

image

**Comment by esarajyan** [comment]
Title and button need to be positioned higher
image

File: frontend/src/text-animation.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    34%  { content: 'Lorem ipsum dolor '; }
+    36%  { content: 'Lorem ipsum dolor s'; }
+    38%  { content: 'Lorem ipsum dolor si'; }
+    40%  { content: 'Lorem ipsum dolor sit'; }
+    42%  { content: 'Lorem ipsum dolor sit '; }
+    44%  { content: 'Lorem ipsum dolor sit a'; }
+    46%  { content: 'Lorem ipsum dolor sit am'; }
+    48%  { content: 'Lorem ipsum dolor sit ame'; }
+    50%  { content: 'Lorem ipsum dolor sit amet'; }
+    52%  { content: 'Lorem ipsum dolor sit amet '; }
```
Comment (esarajyan) [inline frontend/src/text-animation.css:64]: This might be easier to implement in javascript? If not, then this is fine.

**Review by esarajyan — CHANGES_REQUESTED** [review]
I would also move the `.css` and `.js` files into the `static` folder.

**Comment by esarajyan** [comment]
> Remove the `dist` folder. It contains the build of the project, which can be regenerated anytime. It is usually not recommended to push to the repo.

Better yet, add `dist` to `.gitignore`

**Comment by VMFenix8** [comment]
> Space needs to be added here
> 
> image

On my side it looks fine but ill change it and send it in

File: frontend/src/text-animation.css (hunk 1 above)
Comment (VMFenix8) [reply frontend/src/text-animation.css:64]: Ima keep it a stack idk how to change it into javascript if I do figure it out I will do it though

**Review by mgodje — COMMENTED** [review]
old pull request, see "Fixed Features"

**Pull request closed without merging**